### PR TITLE
general clean-up

### PR DESCRIPTION
**Pull request by PascalinDe**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -7,6 +7,25 @@ on:
...
+      - uses: actions/checkout@v2
+      - name: Install linters
+        run: |
+          python -m pip install black flake8
+      - name: Check Python code formatting with Black
+        run: |
+          black --check --quiet .
+      - name: Lint Python code with Flake8
+        run: |
+          flake8
```
Comment (ineiti) [inline .github/workflows/test.yaml:28]:
```suggestion
          flake8

```

File: tools/tests/test_dirinfo.py [hunk 2]
```diff
@@ -260,8 +253,7 @@ def test_select_routers(consensus, vote):
 
 
 def test_generate_signed_consensus_from_real_data(consensus, vote):
-    """
-    """
+    """ """
```
Comment (ineiti) [inline tools/tests/test_dirinfo.py]:
```suggestion
```

Or does that break the test?

**Review by ineiti — CHANGES_REQUESTED** [review]
Who am I to tell the linter what to do?

File: tools/tests/test_dirinfo.py (hunk 2 above)
Comment (PascalinDe) [reply tools/tests/test_dirinfo.py]: no, I think someone just forgot to add a comment there...

**Review by ineiti — APPROVED** [review]

**Pull request merged**